kustomization.yaml

```diff
@@ -10,4 +10,4 @@ configMapGenerator:
 
 images:
 - name: ghcr.io/cybozu-go/accurate
-  newTag: 0.2.0
+  newTag: 0.3.0
```

Comment:
Is this file still referenced in the user manual?
If not, why don't we remove this?

Reply:
I cut corners.
I must release accurate and helm respectively.

Reply:
@zoetrope 
My point is not about Helm.  Can we remove this `kustomizatoin.yaml`?

Reply:
In fact, this is not working already because `config.yaml` is missing.

```yaml
resources:
- config/default

# Edit config.yaml to customize Accurate configurations
configMapGenerator:
- name: accurate-config
  namespace: accurate
  files:
  - config.yaml
```

Reply:
I was mistaken.
`kustomization.yaml` is not used and will be removed.